chapters/envs.rst

```diff
@@ -14,157 +16,168 @@ Here is the list of environment variables that ``jina`` respects during runtime.
         .. highlight:: python
         .. code-block:: python
 
-            os.environ['JINA_LOG_PROFILING'] = 'true'
+            os.environ['JINA_ARRAY_QUANT'] = 'fp16'
 
             from jina.flow import Flow
 
 
+.. confval:: JINA_ARRAY_QUANT
 
-.. confval:: JINA_FULL_CLI
-
-    When set, show all Jina subcommands.
+    Quantization scheme when storing ndarray into protobuf message, useful for reducing the network latency and saving bandwidth. Possible values: ``fp16`` (almost lossless), ``uint8``.
 
     :default: unset
 
-.. confval:: JINA_PROFILING
-
-    Set to any non-empty value to turn on service-level time profiling for Jina.
-
-    :default: unset
+.. confval:: JINA_BINARY_DELIMITER
 
+    The delimiter used in :class:`BasePbIndexer`. Often use a delimiter phrase that you can determine does not occur. (Think of the mime message boundaries)
 
-.. confval:: JINA_WARN_UNNAMED
+.. confval:: JINA_CONTRIB_MODULE
 
-    Set to any non-empty value to turn on the warning for unnamed executors.
+    Paths of the third party components.
 
     :default: unset
 
-.. confval:: JINA_VCS_VERSION
+.. confval:: JINA_CONTRIB_MODULE_IS_LOADING
 
-    Git version of ``jina``. This is used when ``--check-version`` is turned on. For official docker image of ``jina``, ``JINA_VCS_VERSION`` is automatically set to the git version during the building procedure.
+    Set to True when loading the third party components.
 
-    :default: the git head version for ``jina`` image. If you are using ``jina`` locally outside docker container then this is unset.
+    :default: unset
 
 .. confval:: JINA_CONTROL_PORT
 
     Control port of all pods.
 
     :default: unset. A random port will be used for each :func:`BasePod`.
 
+.. confval:: JINA_DEFAULT_HOST
 
-.. confval:: JINA_CONTRIB_MODULE
+    The default host address of Jina.
 
-    Paths of the third party components.
+    :default: `0.0.0.0`
+
+.. confval:: JINA_DISABLE_UVLOOP
+
+    ``uvloop`` is ultra fast implementation of the asyncio event loop on top of libuv. Since Jina 0.3.6, Jina relies on ``uvloop`` to manage the sockets and streams.
+
+    :default: unset
+
+.. confval:: JINA_EXECUTOR_WORKDIR
```

Comment:
is this being used?

Reply:
I think so. I checked with the list in jina/init.py. And this is used in jina/executor/init.py